pulsar-broker/pom.xml

```diff
@@ -406,6 +406,12 @@
       <artifactId>javax.activation</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <scope>test</scope>
+    </dependency>
+
```

Comment:
I see another place modified in the original patch.
```
<relativePath>..</relativePath> to <relativePath>../pom.xml</relativePath>
```

Reply:
Yes. I see only changed this module, so just avoid to cherry-pick it to branch-2.11 for consistency